packages/conventional-commits-parser/test/parser.spec.js

```diff
@@ -285,6 +285,17 @@ describe('parser', function() {
     });
   });
 
+  it('should truncate from scissors line', function() {
+    var msg = parser(
+      'this is some subject before a scissors-line\n' +
```

Comment:
I'd be tempted to assert that the subject line has been populated appropriately (ensuring that your truncation logic wasn't too aggressive).

Reply:
Opted for additional cases here to describe explicity what was tested.